playbooks/provisioner/foreman/pre.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Why do we need the "pre.yaml" here?
Since we don't have a "main.yml" that uses this, please have this in cleanup.yaml